src/api/goods.js

```diff
@@ -3,9 +3,16 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll() {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error(`${response.status} - ${response.statusText}`);
+      }
+
+      return response.json();
+    });
 }
 
-export const get5First = () => {};
+export const get5First = result => result.slice(0, 5);
```

Comment:
you completed only one part of this task (`Load 5 first goods` shows 5 first `goods` after sorting them by name)

Reply:
Thanks! My inattentiveness